src/components/table/table.sandbox.html

```diff
@@ -0,0 +1,65 @@
+<div class="m-table-sandbox">
+    <h2>Classic table</h2>
+    <m-table :columns="simpleColumns" :rows="rows"></m-table>
+
+    <h2>Semi-complex table</h2>
+    <m-table :columns="columns" :rows="rows" style="width:30%;">
+        <template slot="header">
+            <thead>
+                <tr>
+                    <th colspan="2">NAME, USERNAME</th>
+                    <th>AGE</th>
+                </tr>
+            </thead>
+        </template>
+        <template slot="body">
+            <tbody>
+                <tr v-for="row in rows">
```

Comment:
toujours ajouter un :key au v-for , voir-> https://vuejs.org/v2/style-guide/#Keyed-v-for-essential